daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-patterns.md

```diff
@@ -262,6 +262,92 @@ A workflow implementing the monitor pattern can loop forever or it can terminate
 This pattern can also be expressed using actors and reminders. The difference is that this workflow is expressed as a single function with inputs and state stored in local variables. Workflows can also execute a sequence of actions with stronger reliability guarantees, if necessary.
 {{% /alert %}}
 
+## Human interaction
+
+In some cases, a workflow may need to pause and wait for a human to perform some action. For example, a workflow may need to pause and wait for a human to approve a purchase order. Dapr Workflow supports this pattern natively via the [external events]({{< ref "workflow-features-concepts.md#external-events" >}}) feature. Here's an example flow:
```

Comment:
I would move the pub/sub paragraph at the to be at the start with some slight rewording. Pub/sub events from other system, triggering workflows is going to be very common IMO

```suggestion
In some cases, a workflow may need to pause and wait for an external systems to perform some action. External events can be triggered by humans or by other systems.  For example, a workflow may need to pause and wait for a payment to be received. In this case, a payment system might publish an event to a pub/sub topic on receipt of a payment, and a listener on that topic can raise an event to the workflow using the raise event workflow API."

Another very common scenario is when a workflow needs to pause and wait for a human, for example when approving a purchase order. Dapr Workflow supports this event pattern via the [external events]({{< ref "workflow-features-concepts.md#external-events" >}}) feature. 

Here's an example workflow for a purchase order involving a human:
```